neqo-transport/src/connection/tests/mod.rs

```diff
@@ -213,12 +212,7 @@ fn handshake_with_modifier(
             a.test_frame_writer = None;
             did_ping[a.role()] = true;
         }
-        assert!(had_input || output.is_some());
```

Comment:
As discussed before, this is no longer true with the `handshake_delay_with_ecn_blackhole` where initially a server will not receive nor send anything due to the blackhole.